src/ray/test/run_gcs_tests.sh

```diff
@@ -6,7 +6,7 @@
 set -e
 set -x
 
-bazel build "//:redis_gcs_client_test" "//:actor_state_accessor_test" "//:asio_test" "//:libray_redis_module.so"
+bazel build "//:redis_gcs_client_test" "//:actor_state_accessor_test"  "//:subscription_executor_test" "//:asio_test" "//:libray_redis_module.so"
```

Comment:
gcs c++ tests are getting more and more complex. Can you simplify them in the next PR? For example, we can change these `cc_binary` to `cc_test` (we can pass the redis server command as the argument of the `cc_test`). Then we can get rid of this file.

Reply:
It's not urgent so I will do it when the main job of new `GCSClient` finished.
These tests may not run in parallel, otherwise you will need too many redis instances.

Reply:
ok, then can you add a TODO in `build.bazel`?